app/controllers/active_matches_controller.rb

```diff
@@ -18,9 +18,17 @@ def index
     @matches = match_scope
     @current_step = params[:current_step]
     @current_program = params[:current_program]
+    @current_contact_type = params[:current_contact_type]&.to_sym
```

Comment:
`@active_filter` needs to include the new fields in both `closed_matches_controller.rb` and `active_matches_controller.rb` so the filter reflects that it's active when things are chosen.